src/fidocadj/FidoMain.java

```diff
@@ -395,65 +395,61 @@ public CreateSwingInterface()
     @Override
     public void run()
     {
-        /** *****************************************************************
-         * PLATFORM SELECTION AND CONFIGURATION CODE GOES IN THIS SECTION
-         ****************************************************************** */
-        if (OSValidator.isMac()) {
-            AccessResources g = new AccessResources();
+        SettingsManager settingsManager = new SettingsManager(this.getClass());
+        boolean enableCustomThemes = settingsManager.get("ENABLE_CUSTOM_THEMES",
+                "false").equals("true");
+        String theme = settingsManager.get("THEME", "light");
+        boolean isLightTheme = theme.equals("light");
+        boolean isDarkTheme = theme.equals("dark");
 
-            SettingsManager settingsManager = 
-                                new SettingsManager(AccessResources.class);
+        try {
+            if (enableCustomThemes) {
+                applyTheme(isLightTheme, isDarkTheme);
+            }
+        } catch (Exception e) {
+            System.out.println("Failed to apply theme. Falling back to default.");
```

Comment:
Should we write on the error stream for a message like this?

Reply:
Maybe so, but I'd still like to review the entire exception handling system if you agree. Take a look here: https://github.com/DarwinNE/FidoCadJ/discussions/254